### DISCUSSION TITLE
Adding Permanent Electronic Meetings

### DISCUSSION THREAD

**Discussion by VickiB11** [code4recovery/12-step-meeting-list#939] [Q&A] [answered]
I am the Group Records Coordinator for an Al-Anon Area.  I use the TSML meeting finder on our area website. The Area will be voting on adding meetings that meet only electronically, Permanent Electronic Meetings, to the area service structure.  I believe the motion will pass. Since these groups have no physical meeting space, not even a city or location, how will I be able to add these PEMs to the TSML meeting finder? 

Vicki Bauer

**Comment by joshreisner** [comment, marked as answer]
Hi Vicki,

Every meeting has some relationship to geography by virtue of having a timezone. Further, most sites have some level of geographic focus. 

We recommend you continue adding approximate locations to these meetings. These can be very broad, such as a city or a state. Often this is the place of origin of a meeting. Anywhere within the meeting’s timezone.

**Reply by VickiB11** [reply]
That worked in September, but now it doesn't.  I can create and post the meeting, but it is labeled Inactive.
![PEG listed as inactive](https://user-images.githubusercontent.com/74988606/209584806-954eee91-f4fa-48b0-b25d-1d3430b8cf20.png)

**Reply by joshreisner** [reply]
@VickiB11 i dont see a conference url or phone in the screenshot - is there one?

**Comment by VickiB11** [comment]
Josh, thank you for pointing that out. It’s been a minute since I created the first test listing in September. I forgot to add the url this time. I added it and it works now. Thank you for your feedback!On Dec 26, 2022, at 6:36 PM, Josh Reisner ***@***.***> wrote:﻿
@VickiB11 i dont see a conference url or phone in the screenshot - is there one?
